grails-app/services/org/pih/warehouse/tableroapi/NumberDataService.groovy

```diff
@@ -249,4 +251,34 @@ class NumberDataService {
             subTitle, "/openboxes/inventory/listExpiredStock?status=expired"
             )
     }
+
+    NumberData getOpenStockRequests(def location) {
+        def openStockRequests = Requisition.executeQuery("""
+            SELECT COUNT(distinct r.id) FROM Requisition r
+            WHERE r.origin = :location
+            AND r.sourceType = :sourceType
+            AND r.status NOT IN (:statuses)
+            """,
+                [
+                        'location': location,
+                        'sourceType' : RequisitionSourceType.ELECTRONIC,
+                        'statuses' : [RequisitionStatus.CREATED, RequisitionStatus.ISSUED, RequisitionStatus.CANCELED],
+                ])
+
+        def title = [
+                code : "react.dashboard.numberData.openStockRequests.label",
+                message : messageService.getMessage("react.dashboard.numberData.openStockRequests.label")
+        ]
+
+        def subTitle = [
+                code : "react.dashboard.subtitle.requests.label",
+                message : messageService.getMessage("react.dashboard.subtitle.requests.label")
```

Comment:
See discussion in this ticket (https://pihemr.atlassian.net/browse/OBPIH-3416) for deciding on a consistent localization key for these messages. Please comment in the ticket and let's discuss during tech huddle on Friday. For this one, let's have a quick discussion here to try to pin down generally what we want. I feel like subtitle before requests doesn't make sense. Also, requests by itself is too general, so let's give it a more specific id like openRequests or openStockRequests.
```
react.dashboard.requests.subtitle.label
```

Reply:
@jmiranda I agree that removing subtitle is a good idea but in this case requests is not too general because this label literally says "Requests". We also have label for openStockRequests.

Reply:
@jmiranda So I will change this label to
`react.dashboard.requests.subtitle.label`
and `react.dashboard.numberData.openStockRequests.label`
to` react.dashboard.openStockRequests.title.label`.
Is that fine?

Reply:
@DominikaHatala Yes. Let's follow the ￼`react.dashboard.<indicatorId>.<otherElementType>.labe￼l` pattern.

Reply:
@jmiranda @awalkowiak Done.